### PR TITLE
Solved recent TS retrieval failure, added test to verify functionality

### PR DESCRIPTION
**Pull request by zack-rma**
Addresses Issue #874 
Solved recent TS retrieval failure, added test to verify functionality.

### CODE REVIEW

**Review by MikeNeilson — APPROVED** [review]

**Comment by MikeNeilson** [comment]
Need to merge in https://github.com/USACE/cwms-data-api/pull/882 first. Our github actions got rather behind and finally deprecated.

**Comment by MikeNeilson** [comment]
Or you can rebase from and submit the PR to update_dev/from_release.

**Comment by MikeNeilson** [comment]
@zack-rma please merge/rebase this so that the builds can run properly.

**Comment by zack-rma** [comment]
Rebased to fix build errors

File: cwms-data-api/src/test/java/cwms/cda/api/TimeSeriesRecentControllerIT.java [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+
+@Tag("integration")
+class TimeSeriesRecentControllerIT extends DataApiTestIT {
+    TestAccounts.KeyUser user = TestAccounts.KeyUser.SPK_NORMAL;
+    private static final Logger LOGGER = Logger.getLogger(TimeSeriesRecentControllerIT.class.getName());
+    private static final String OFFICE_ID = "SPK";
+    private static final String LOCATION = "ZACK1";
+    private static final String TS_ID = LOCATION + ".Depth.Inst.15Minutes.0.TS_TEST";
+    private static final String CATEGORY_ID = "TEST";
```
Comment (MikeNeilson) [inline cwms-data-api/src/test/java/cwms/cda/api/TimeSeriesRecentControllerIT.java]: The location name itself is fine, please use a more "real" name for the category and group.

**Review by MikeNeilson — CHANGES_REQUESTED** [review]
Nit pick about group/category names.

Also the integration tests is failing. Not super obvious why just looking at the output (just no results returned.)

File: cwms-data-api/src/test/java/cwms/cda/api/TimeSeriesRecentControllerIT.java (hunk 1 above)
Comment (zack-rma) [reply cwms-data-api/src/test/java/cwms/cda/api/TimeSeriesRecentControllerIT.java]: Replaced with more realistic naming. 

Also fixed the integration test, had to swap from hard-coded dates to using now() for ensuring TS dates are within recent time window

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**